core/src/main/java/tech/pegasys/web3signer/core/Runner.java

```diff
@@ -125,6 +127,17 @@ public void run() {
               "signers_loaded_count",
               "Number of keys loaded (combining SECP256k1 and BLS12-381");
 
+      final SlashingProtection slashingProtection;
+      if (config.isSlashingProtectionEnabled()) {
+        slashingProtection =
+            DbBackedSlashingProtectionFactory.createDbBackedSlashingProtection(
+                config.getSlashingStorageUrl(),
+                config.getSlashingStorageUsername(),
+                config.getSlashingStoragePassword());
+      } else {
+        slashingProtection = null;
```

Comment:
Would rather a noop slashingProtection than to pass in a null

Reply:
done